board/Alix/setup.sh

```diff
@@ -25,7 +25,7 @@ alix_partition_image ( ) {
         # TODO: This is broken; should use 'make install' to copy
         # bootfiles to workdir, then install to disk image from there.
```

Comment:
It would be better to fix this todo and use installed files not files plucked from objdir.